platform/broadcom/sonic-platform-modules-micas/common/modules/linux-5.10/wb_tps53622.c

```diff
@@ -0,0 +1,265 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
master image is already kernel 6.1 iirc, why still try to do linux 5.10?

Reply:
Yea, it is simply a name issue, will not affect compiling  or functionality, since this pr was raised long time ago when the kernel is still 5.10. Will rename it on next phase.